src/pages/Lesson10.tsx

```diff
@@ -0,0 +1,95 @@
+import { FC } from 'react';
+import AppWrapper from '../components/AppWrapper';
+import NavButtons from '../components/NavButtons';
+import Terminal from '../components/Terminal';
+import { HeaderSections } from '../types/globalTypes';
+
+const Lesson10: FC = () => {
+  const code1 = `
+    class Node
+    {
+        int value;
+        Node* next;
+        Node(int val){value = val;}
+    };
+    `;
+
+  const code2 = `
+    Node* newNode = new Node(1); //make new node with
+    cout << newNode ->value << endl; //prints 1
```

Comment:
Sorry, realized the Figma was incorrect here. can you make these small updates to the text here?
<img width="597" alt="image" src="https://github.com/uclaacm/parcel-pointers/assets/78634083/2c20bb91-91aa-4285-b1a1-f0f602412fc6">